client/my-sites/customize/main.jsx

```diff
@@ -1,83 +1,96 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	url = require( 'url' ),
-	Qs = require( 'qs' ),
-	cloneDeep = require( 'lodash/cloneDeep' ),
-	connect = require( 'react-redux' ).connect,
-	debug = require( 'debug' )( 'calypso:my-sites:customize' );
-import { get, startsWith } from 'lodash';
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import url from 'url';
+import Qs from 'qs';
+import cloneDeep from 'lodash/cloneDeep';
+import get from 'lodash/get';
+import startsWith from 'lodash/startsWith';
```

Comment:
I guess `babel-plugin-lodash` takes care of doing this automatically, so I'll use the shorter version.